fitbenchmarking/fitting_benchmarking.py

```diff
@@ -154,45 +154,45 @@ def do_fitting_benchmark_one_problem(prob, minimizers, use_errors=True, count=0,
 
     # Get function definitions for the problem - one for each starting point
     function_defs = get_function_definitions(prob)
-    # search for lowest chi2
-    min_sum_err_sq = 1.e20
+
     # Loop over the different starting points
     for user_func in function_defs:
```

Comment:
Looking at this code in more detail. Can you confirm this is a loop over different starting points. Asking before this is not clear from 'for user_func in function_defs'

Reply:
Yes, this loop over different starting points. Tested it in one of the FittingBenchmarkOneProblem tests.